app/controllers/application_controller.rb

```diff
@@ -11,6 +11,7 @@
 #   CONDITIONS OF ANY KIND, either express or implied. See the License for the
 #   specific language governing permissions and limitations under the License.
 # ---  END LICENSE_HEADER BLOCK  ---
+require 'avalon/errors'
```

Comment:
@cjcolvar This line gives me an error, `cannot load such file -- avalon/errors`.

Reply:
:+1: It worked for me at first but then failed as well.  I'll fix that.

Reply:
It would probably help if I committed that file! 😄 
I pushed it and now the circleci tests are running.